app/controllers/follows_controller.rb

```diff
@@ -53,16 +53,14 @@ def bulk_show
   def create
     authorize Follow
 
-    followable = case params[:followable_type]
-                 when "Organization"
-                   Organization.find(params[:followable_id])
-                 when "Tag"
-                   Tag.find(params[:followable_id])
-                 when "Podcast"
-                   Podcast.find(params[:followable_id])
-                 else
-                   User.find(params[:followable_id])
-                 end
+    followable_klass = case params[:followable_type]
```

Comment:
I think it's good idea to have some redundancy in place since this is one of the area that was part of the problem

```suggestion
    followable_klass = case params[:followable_type].capitalize
```